system/mp/mpcomm.cpp

```diff
@@ -1801,7 +1823,7 @@ CMPConnectThread::CMPConnectThread(CMPServer *_parent, unsigned port)
             port = minPort + getRandom() % numPorts;
             try
             {
-                listensock = ISocket::create(port, 16);  // better not to have *too* many waiting
+                listensock = ISocket::create(port, 800);
```

Comment:
I will rework to try and account for concurrent connections.  I do not want to warn always for single node and small configs.